src/lakefs_spec/spec.py

```diff
@@ -33,6 +33,30 @@
 EmptyYield = Generator[None, None, None]
 
 
+class LakectlConfig(NamedTuple):
+    host: str | None = None
+    username: str | None = None
+    password: str | None = None
+
+    @classmethod
+    def read(cls, path: str | Path) -> "LakectlConfig":
+        try:
+            import yaml
+        except ModuleNotFoundError:
+            return cls()
```

Comment:
Should we have some kind of information or print when the config file is not read because of missing `PyYaml` dependency? 

We should also specify this dependency in the readme.

Reply:
Good idea. The dependency extra was added to the readme in [486e183](https://github.com/appliedAI-Initiative/lakefs-spec/pull/45/commits/486e1833f0d1ec48a0d5ea8ea2bda2ef05d53c80).

Reply:
Warning was added in [4e26d26](https://github.com/appliedAI-Initiative/lakefs-spec/pull/45/commits/4e26d2693a54cb96c1c85072d6b9aa43d65b3f80).